ToukMultiplex/src/main/java/com/example/toukmultiplex/Reservation/AmountToPayService.java

```diff
@@ -0,0 +1,38 @@
+package com.example.toukmultiplex.Reservation;
+
+import com.example.toukmultiplex.Config.TicketProperties;
+import com.example.toukmultiplex.Reservation.DTO.ReservationDataDto;
+import com.example.toukmultiplex.Reservation.DTO.SeatDTO;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Service
+class AmountToPayService {
+
+    private final TicketProperties ticketProperties;
+
+    public AmountToPayService(TicketProperties ticketProperties) {
+        this.ticketProperties = ticketProperties;
+    }
+
+    BigDecimal amountToPay(ReservationDataDto reservationDataDto) {
```

Comment:
Czy tą metodę dałoby się zaimplementować przy pomocy streamów?

Reply:
Nie będę ściemniał, z tego co zerknąłem w internecie to tak, ale ja jeszcze nie potrafię